filebeat/filebeat.full.yml

```diff
@@ -395,22 +395,206 @@ output.elasticsearch:
   # Configure curve types for ECDHE based cipher suites
   #tls.curve_types: []
 
+#------------------------------- Kafka output ---------------------------------
+#output.kafka:
+  # The list of Kafka broker addresses to connect to.
+  #hosts: ["localhost:9092"]
```

Comment:
Changing it to "The list of Kafka broker addresses from where to fetch the cluster metadata.". What do you say, @urso?